tasks/condition/condition.js

```diff
@@ -39,6 +68,18 @@ const operator = "add";
 const redemption = 420;
 let discount;
 
+if (redemption >=0 && redemption <= 350) {
+    discount === 0;
```

Comment:
=== is a comparison operator, but you need to make an assignment here, so:
````
discount = 0;
````

Please do that for other lines as well